### ISSUE TITLE
aldryn-blog and aldryn-newsblog should be able to coexist in the same project

### ISSUE THREAD

**Issue by stefanfoulis** [aldryn/aldryn-newsblog#90]
currently there are foreign key reverse naming problems

- [x] use the ``cmsplugin_ptr = models.OneToOneField(CMSPlugin, related_name='+', parent_link=True)`` trick on plugin models @stefanfoulis
- [x] unique plugin names in cms_plugins.py (prefix with app name)

**Comment by stefanfoulis** [comment]
can be solved by defining this on the plugin model:
```python
cmsplugin_ptr = models.OneToOneField(CMSPlugin, related_name='+', parent_link=True)
```
(as long as the Plugin Name is unique)

**Comment by mkoistinen** [comment]
This and #122 and #126 should be fixed at the same time.

**Comment by mkoistinen** [comment]
Closed by #128
